Contuity/Views/Explore/ExplorePresenter.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ExplorePresenter.swift
+//  Contuity
+//
+//  Created by Anthony on 10/18/18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+import Foundation
+
+protocol ExplorePresenterProtocol: PresenterProtocol {
+    /// The jots that need to be displayed
+    var jots: [Jot] { get set }
+}
+
+class ExplorePresenter: ExplorePresenterProtocol {
+    var jots: [Jot] = []
+    init() {
+        print("hi")
```

Comment:
why is this init() necessary (I could be missing something)?

I believe you could move the jots functionality into the var.

`var jots: [Jot] { try? Jot.getJots(queue: false) ?? []}`

Reply:
getJots can throw an error and you can't do error handling when initializing a property variable so you have to do it in init()

Reply:
Really? You can't just use `try?`? Weird